_targets.R

```diff
@@ -26,7 +26,18 @@ list(
   # PULL SITE DATA
   tar_map(
     values = tibble(state_abb = states),
-    tar_target(nwis_data, get_site_data(oldest_active_sites, state_abb, parameter))
+    tar_target(nwis_inventory,
+               {
+                 oldest_active_sites %>%
+                   filter(state_cd == state_abb)
+               }
```

Comment:
Not a suggestion for a change, but a fun thing to know: since your only code in the command is a piped series of lines, you don't need to wrap them in `{ }`. You can simply put 

```r
tar_target(nwis_inventory,
           oldest_active_sites %>%
               filter(state_cd == state_abb)
),
```

Reply:
Oh good to know - I was wondering about that exact thing